docs/Course Wiki/CS Course Offerings/CS374A.md

```diff
@@ -0,0 +1,95 @@
+# CS374A
+
+## Content Covered
+
+### First third (languages)
+
+- Induction, with a focus on application to strings
+- Regular languages
+- Regular expressions
+- Deterministic finite autoamta (DFAs)
+- Nondeterminstic finite automata (NFAs)
+- Regular language transformation via DFA and NFA manipulations
+- Myhill-Nerode theorem and proving irregularity with fooling sets
+- Context free languages and context free grammars (CFGs)
+- Turing machines (only some semesters)
+
+### Middle third (algorithms)
+
+- Divide and conquer algorithms
```

Comment:
Add MSTs? Or maybe expand on what graph algorithms are covered.

Reply:
Will add MSTs. There's a lot of algos covered so I'll just reword to "graph search algorithms"